QUARKUS-713.md

```diff
@@ -0,0 +1,46 @@
+# QUARKUS-713 Tech preview support for the OpenShift Client
+
+This document describes a high-level testing overview of [QUARKUS-713 Tech preview support for the OpenShift Client](https://issues.redhat.com/browse/QUARKUS-713).
+
+The goal of QUARKUS-713 is to provide the users with tech preview support of utility that will expose an OpenShift
+client. This utility will make it possible for the users to develop their kubectl plugins or OpenShift operators.
+
+## Scope of the testing
+### Smoke testing
+[`OpenShiftClientTest`](https://github.com/quarkusio/quarkus/blob/master/integration-tests/openshift-client/src/test/java/io/quarkus/it/openshift/client/OpenShiftClientTest.java) 
+was implemented as part of feature development. This unit test verifies the basic functional assertions in a mock 
+environment.
+
+This test is executed upstream as a part of Quarkus test suite.
+
+### Component testing
+The feature itself consists of exposing fabric8 OpenShift client to the Quarkus application conveniently.
+
+fabric8 itself has an [upstream CI](https://github.com/fabric8io/kubernetes-client/actions) where the functionality of 
+all the client libraries is verified.
+
+### Integration testing
+An integration test verifying correctness of the feature on OpenShift clusters will be implemented in 
+[quarkus-qe/quarkus-openshift-test-suite](https://github.com/quarkus-qe/quarkus-openshift-test-suite).
+
```

Comment:
Please add some details about current usage of `OpenShiftClient` in OpenShiftTS (outside the OpenShift cluster scenario).

Add explicit note that the new test coverage is about functionality of `OpenShiftClient` inside the OpenShift cluster.